safenode/src/client/chunks/mod.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+mod error;
+mod pac_man;
+
+pub(crate) use self::error::{Error, Result};
+pub(crate) use pac_man::{encrypt_large, to_chunk, DataMapLevel};
+
+use bytes::Bytes;
+use self_encryption::MIN_ENCRYPTABLE_BYTES;
+
+/// Data of size more than 0 bytes less than [`MIN_ENCRYPTABLE_BYTES`] bytes.
+///
+/// A `Spot` cannot be self-encrypted, thus is encrypted using the client encryption keys instead.
+#[allow(missing_debug_implementations)]
+#[derive(Clone)]
+pub(crate) struct SmallFile {
+    bytes: Bytes,
```

Comment:
`Spot` here in the comment is old nomenclature isn't it?

Reply:
Yeah definitely. Will fix!